code/modules/shuttle/mini_dropship.dm

```diff
@@ -185,6 +185,12 @@
 		shuttle_port.set_idle() // don't go up with a broken console, cencel spooling
 		visible_message("Autopilot detects loss of helm control. Halting take off!")
 
+/obj/machinery/computer/camera_advanced/shuttle_docker/minidropship/attack_hand(mob/user)
```

Comment:
use can_interact proc.

The rest of the code will be similar

Reply:
done.